### PR TITLE
removed one question from mentor application and re-worded remaining …

### PR DESCRIPTION
**Pull request by VABIII**
Removed one question from careers portion of the mentor application form, and re-worded the remaining questions as described in the google doc that is attached to the Trello card for this pull request. Also, I changed the tech stack question from a dropdown to a group of checkboxes as well.

#### Video Link

[Loom Video](https://www.loom.com/share/c9fc850c9f0b43de85e153cf02032a20)

#### Trello Link

<blockquote class="trello-card"><a href="[https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JfipwNkb&#x2F;265-fix-refactor-sidebar-to-have-icons-representing-each-element-in-the-sidebar](https://trello.com/c/k2Mffclg/360-mentor-intake-what-do-you-able-to-contribute-to-underdogdev-section">Fix: Mentor intake what are you able to contribute to underdogdev-section</a></blockquote>

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by mikayla-kosmala — APPROVED** [review]
Looks good! Nice turn around!

File: src/components/pages/Navbar/Navbar.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
               </div>
             )}
 
-            {isAuthenticated && (
+            {!isAuthenticated && (
```
Comment (benedictSchurwanz) [inline src/components/pages/Navbar/Navbar.js:113]: This makes sense - showing the buttons if the user is not logged in. Though I'm not sure why it was in this PR, it doesn't seem related to the other changes, and it wasn't mentioned in your video. I guess that's a little bug fix.

**Review by benedictSchurwanz — COMMENTED** [review]
See my comments on the Navbar.js changes

**Review by benedictSchurwanz — APPROVED** [review]
